phpBB/phpbb/console/command/user/delete_id.php

```diff
@@ -170,4 +197,25 @@ protected function interact(InputInterface $input, OutputInterface $output): voi
 			}
 		}
 	}
+
+	/**
+	 * Deletes a bot user
+	 *
+	 * @param array $user_row
+	 * @return void
+	 */
+	protected function delete_bot_user(array $user_row): void
+	{
+		$sql = 'DELETE FROM ' . $this->bots_table . '
+					WHERE user_id = ' . (int) $user_row['user_id'];
+		$this->db->sql_query($sql);
```

Comment:
This looks identical to the query below so probably could be added to the loop.

Reply:
Good point, was based on acp_bots but since we're doing something slightly different here we don't need to separate this.